packages/third-parties/README.md

```diff
@@ -0,0 +1,44 @@
+# `@next/third-parties`
+
+`@next/third-parties` is collection of components that can be used to efficiently load third party libraries into your Next.js application.
+This package uses [Third Party Capital](https://github.com/GoogleChromeLabs/third-party-capital/) under the hood to fetch the best loading recommendations.
+
+> Note: `@next/third-parties` is still experimental and under active development
+
+# Usage
+
+## Google third parties
+
+### Youtube Embed
+
+The `YoutubeEmbed` component is used to load and display a YouTube embed. This component loads approximately 224× faster by using [lite-youtube-embed](https://github.com/paulirish/lite-youtube-embed) under the hood.
```

Comment:
Let's remove the 224x number (I recognize it's taken from the lite-youtube-embed docs, but we don't have a way to review the methodology)

```suggestion
The `YoutubeEmbed` component is used to load and display a YouTube embed. This component loads faster by using [lite-youtube-embed](https://github.com/paulirish/lite-youtube-embed) under the hood.
```